src/inference/tests/unit/cpu_map_parser.cpp

```diff
@@ -0,0 +1,571 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <common_test_utils/test_common.hpp>
+
+#include "ie_system_conf.h"
+
+using namespace testing;
+using namespace InferenceEngine;
+
+namespace {
+
+#ifdef __linux__
+
+struct LinuxCpuMapTestCase {
+    int _processors;
+    int _sockets;
+    int _cores;
+    std::vector<std::vector<int>> _proc_type_table;
+    std::vector<std::vector<int>> _cpu_mapping_table;
+    std::vector<std::vector<std::string>> system_info_table;
+};
+
+class LinuxCpuMapParserTests : public CommonTestUtils::TestsCommon,
```

Comment:
Will you create same tests for Windows?

Reply:
Yes, will create tests for windows as well.